pkg/operator/controller/status.go

```diff
@@ -21,10 +21,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	IngressClusterOperatorName = "ingress"
```

Comment:
I thought the request was to drop the `openshift-` prefix — wouldn't this then become `ingress-operator`?

Reply:
The referenced Slack message says to drop both.